FlaskApp/tests/test_rideoffers.py

```diff
@@ -0,0 +1,46 @@
+import unittest
+from FlaskApp.api.rideoffers import RideOffer
+
+class RideOfferTestCase(unittest.TestCase):
```

Comment:
I don't think these tests are of relevance since you are already testing the endpoints so this should be covered. You are just repeating yourself.